### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by nicholaspulido**

### CODE REVIEW

**Comment by nicholaspulido** [comment]
Please approve PER @ckilpatrick20

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+FREE 99
```
Comment (craigkilpatrick) [inline LICENSE:1]: Hey @nicholaspulido why did you add this line?

Comment (nicholaspulido) [reply LICENSE:1]: How else will people know what license this sweet code is under?

**Review by craigkilpatrick — APPROVED** [review]
Looks Good!

**Comment by nicholaspulido** [comment]
k
